src/Application/Adapters/Laravel/Commands/InsightsCommand.php

```diff
@@ -36,6 +37,7 @@ public function handle(): int
         }
 
         $configuration = require $configPath;
+        $configuration['fix'] = $this->input->hasOption('fix') && (bool) $this->input->getOption('fix') === true;
```

Comment:
Laravel has some methods to shorten this.
```suggestion
        $configuration['fix'] = $this->hasOption('fix') && (bool) $this->option('fix') === true;
```

Reply:
Didn't know that! Thanks :slightly_smiling_face: